specification.md

```diff
@@ -133,122 +153,130 @@ Example meta file for an Entry:
     uuid: b05c865d-fb68-44de-86fc-1e95b273159c
     animal: bk196
 
-Entry directories must not contain root or entry directories.
-Any subdirectories are ignored.
-
 ### Datasets
 
-A *dataset* is defined as a concrete time series or point process.  Multiple
-datasets may be stored in an entry, and may be unequal in length or have
-different *timebases*.
-
-Attributes are stored in a YAML file with `.meta` appended to the dataset name.
-
-A *timebase* is defined by two quantities (with units), one of which is optional
-under some circumstances. The required quantity is the *offset* of the data.
-All time values in a dataset are relative to this time.  The default offset of
-a dataset is the timestamp of the entry.  Individual datasets may have their
-own offsets, which are calculated relative to the entry timestamp.
-
-The second quantity in a timebase is the *sampling rate*, which allows discrete
-times to be converted to real times. It is required if the data are sampled (as
-in a time series) or if time values in a point process are in units of samples.
-Only point processes with real-valued units of time may omit the sampling rate.
-
-Real-valued times must be in units of seconds. Discrete-valued times must be in
-units of samples.
+A *dataset* is a concrete time series or point process.  Multiple datasets may
+be stored in an entry, and may have different lengths or timebases (i.e.,
+*offset* and *sampling rate*).
 
 #### Sampled data
 
 Sampled data are stored in raw binary files as outlined in
 [the Neurosuite documentation](http://neurosuite.sourceforge.net/formats.html).
 
-For multi-channel files, samples are interleaved. A raw binary file with N channels and M samples looks like this:
-
-    c1s1, c2s1, c3s1, ..., cNs1, c1s2, c2s2, c3s2, ..., c1sM, c2sM, c3sM,...,cNsM 
-
+These data shall be represented as 2-dimensional arrays of scalar values,
```

Comment:
This is true in memory, but the file itself is a flat binary file with no explicit dimensionality.